workflows/charts/training/huggingface_llm/tests/distilgpt2_values.yaml

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metadata:
+  namespace: kubeflow
+
+secret:
+  encodedToken:
+
+image:
+  name:
```

Comment:
I have the the image name/tag here blank since there's no dockerhub version of this image yet. I tested it manually with a image that I build/pushed to caas.